packages/hw-app-ckb/examples/ckb-sdk-js-example.js

```diff
@@ -0,0 +1,132 @@
+/* eslint-disable */
+const util = require('util')
+const CKB = require('../lib').default
+const formatter = require('../../ckb-sdk-rpc/lib/paramsFormatter').default
+
+// Note: there is currently a speculos bug that appears not to have been
```

Comment:
This note should be written in such a way that someone trying to use this app, who doesn't know at all what Speculos is, does not think there's a serious bug.

Reply:
Updated to be clearer on what useSpeculos is/does to make the caveat make more sense.